src/components/App.vue

```diff
@@ -295,30 +295,39 @@ module.exports = {
 		that.$store.commit("SET_NETWORK", network);
 	    });
 	},
-
 	checkIfDomainNeedsUnblocking() {
 	    if (this.network == null) return;
 	    var that = this;
 	    this.network.otherDomain().thenApply(function (domainOpt) {
-		if (domainOpt.isPresent()) {
+            if (domainOpt.isPresent()) {
+                let domain = domainOpt.get();
+                that.sendNotABlockRequest(domain, (errCb1) => {
+                    that.sendNotABlockRequest(domain, (errCb2) => {
```

Comment:
We should only retry if a http 503 was returned.

Reply:
have added code to detect. needs to be deployed to test env to confirm behaviour

Reply:
Can we also use recursion rather than 3 nested calls? Just have an extra parameter remainingAttempts?